### PR TITLE
[CLS-56454] scripts: add script to clean the persistent directory

### PR DESCRIPTION
**Pull request by oded-s1**
running a simple daemonset with mount to the host, check if the dir exists and delete it.

### CODE REVIEW

File: scripts/k8s_persistent_dir_cleaner.yaml [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+    matchLabels:
+      app: nginx
+  template:
+    metadata:
+      labels:
+        app: nginx
+    spec:
+      containers:
+      - name: nginx
+        image: nginx:latest
```
Comment (natishauli) [inline scripts/k8s_persistent_dir_cleaner.yaml:18]: why nginx?

File: scripts/k8s_persistent_dir_cleaner.yaml [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+        command: ['sh', '-c', 'if [ -d "/host/var/lib/sentinelone" ]; then echo "deleting persistent dir" && rm -rf /host/var/lib/sentinelone; else echo "persistent dir already deleted"; fi && sleep infinity']
```
Comment (natishauli) [inline scripts/k8s_persistent_dir_cleaner.yaml:19]: The persistent dir is configurable except for autopilot, so maybe we should get the path from "{{ include "persistentDir" . }}" ?

File: scripts/k8s_persistent_dir_cleaner.yaml [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+        volumeMounts:
+        - name: host
+          mountPath: /host
+          mountPropagation: HostToContainer
+        securityContext:
+          privileged: true
```
Comment (natishauli) [inline scripts/k8s_persistent_dir_cleaner.yaml:25]: would it run in openshift as well with privileged?

File: scripts/k8s_persistent_dir_cleaner.yaml (hunk 1 above)
Comment (oded-s1) [reply scripts/k8s_persistent_dir_cleaner.yaml:18]: it's for internal use so it doesn't really matter but what do you suggest?

File: scripts/k8s_persistent_dir_cleaner.yaml (hunk 2 above)
Comment (oded-s1) [reply scripts/k8s_persistent_dir_cleaner.yaml:19]: I didn't want to use another values.yaml or to install using helm

File: scripts/k8s_persistent_dir_cleaner.yaml (hunk 3 above)
Comment (oded-s1) [reply scripts/k8s_persistent_dir_cleaner.yaml:25]: I will check but it's for internal use

**Review by natishauli — APPROVED** [review]

**Pull request closed without merging**